### DISCUSSION TITLE
Load Vidéos/Images

### DISCUSSION THREAD

**Discussion by Looxor** [visgl/loaders.gl#2368] [Ideas]
Hi 👋

**Comment by ibgreen** [comment]
@Looxor Thanks for bringing this to my attention.

It looks like at least the AVIF image format is already supported by most browsers (https://caniuse.com/avif), we could make sure the `ImageLoader` can handle it. 

The AV1 video format however seems not supported on Safari: https://caniuse.com/?search=AV1, which probably holds back adoption for now?

If we do add official AVIF support, in the spirit of loaders.gl, we want to make sure all loaders also work under Node.js (so we can't just rely on browsers implementations). This library looks like something we could use for that: https://github.com/jamsinclair/jSquash - perhaps we could also use that to replace the aging polyfills for JPG and PNG

Do you have any specific use cases which we should consider, any preferences on how you would like the AVIF support to work, any issues you are running into right now?

**Reply by ibgreen** [reply]
Thanks for researching. We have been using `sharp` for Node.js image processing in some projects but lately it has not kept pace with Apple M1 silicon which has caused a lot of trouble on new MacBooks.

I note that the Squash library is web assembly instead of linking to native code, so it should always work even on new OS/hardware (but perhaps not quite as performantly).
